.github/workflows/ci.yml

```
@@ -142,7 +142,47 @@ jobs:
           private-github-key: ${{ secrets.PRIVATE_GITHUB_KEY }}
 
       - name: Run live dev env tests
-        run: docker exec tools_awx_1 /bin/bash -c "make live_test"
+        id: make-run
+        run: docker exec tools_awx_1 /bin/bash -c "GITHUB_ACTIONS=${GITHUB_ACTIONS} GITHUB_OUTPUT=${GITHUB_OUTPUT} PYTEST_ADDOPTS='--cov=awx --cov-report=xml --junitxml=../../../../reports/junit.xml --no-cov-on-fail' make live_test"
```

This will probably need to be
```suggestion
        run: docker exec tools_awx_1 /bin/bash -c "GITHUB_ACTIONS=${GITHUB_ACTIONS} GITHUB_OUTPUT=${GITHUB_OUTPUT} PYTEST_ADDOPTS='--cov --cov-config=../../../../.coveragerc --cov-report=xml:../../../../reports/coverage.xml --junitxml=../../../../reports/junit.xml --no-cov-on-fail' make live_test"
```